src/core/radio/radio.hpp

```diff
@@ -408,11 +408,24 @@ class Radio : public InstanceLocator, private NonCopyable
     /**
      * This method updates the CSL sample time in radio.
      *
-     * @param[in]  aCslSampleTime  The CSL sample time.
+     * @param[in]  aCslSampleTime   The CSL sample time.
```

Comment:
nit: unnecessary change?
```suggestion
     * @param[in]  aCslSampleTime  The CSL sample time.
```